myning/chapters/tutorial.py

```diff
@@ -149,5 +151,5 @@ def learn_bindings():
 
 def exit_tutorial():
     game._state = GameState.READY  # pylint: disable=protected-access
-    FileManager.multi_save(game, player, *player.inventory.items)
+    FileManager.multi_save(game, player, *inventory.items)
```

Comment:
interesting. good catch.  this was a find and replace mistake.  I'm not sure what `*inventory.items` is even saving here. I think we should be saving `inventory` instead.

Reply:
we need to save the items as well as the inventory, because if we only save the inventory, exit the game and come back, the inventory will have 3 references to items that don't exist (because each item has its own file). should probably be tested to double check to see what happens if you only save one and not the other

Reply:
Yeah the inventory works the way entities used to by just holding references to the objects on save and loading them from the entities folder on startup, so we'll need both the items and the inventory saved here.

Reply:
Hmm. I see. I'll review the code with this understanding and see what else I missed.  I might implement something in the Inventory class to make it happen when I save Inventory.